core/templates/pages/exploration-editor-page/modal-templates/exploration-metadata-modal.component.spec.ts

```diff
@@ -120,6 +120,52 @@ describe('Exploration Metadata Modal Component', () => {
       fixture.detectChanges();
     });
 
+    it('should validate exploration editor tags properly', fakeAsync(() => {
```

Comment:
Please rephrase the test description to better explain what is baing test from the end-user perspective.

Reply:
done